SmartDeviceLinkTests/SDLStreamingVideoScaleManagerSpec.m

```diff
@@ -40,22 +41,24 @@
     context(@"test scaling a frame", ^{
          it(@"should scale the frame correctly with a scale > 1", ^{
              videoScaleManager.scale = 1.25;
-             CGRect expectedRect = CGRectMake(0, 0, 160, 320);
-             CGRect testRect = videoScaleManager.appViewportFrame;
+             const CGRect expectedRect = CGRectMake(0, 0, 160, 320);
+             const CGRect testRect = videoScaleManager.appViewportFrame;
              expect(@(CGRectEqualToRect(expectedRect, testRect))).to(beTrue());
          });
 
-        it(@"should not scale the frame with a scale < 1", ^{
-            videoScaleManager.scale = 0.3;
-            CGRect expectedRect = CGRectMake(0, 0, testScreenSize.width, testScreenSize.height);
-            CGRect testRect = videoScaleManager.appViewportFrame;
+        it(@"should scale the frame correctly with the scale 0.3", ^{
+            const float scale = 0.3f;
+            videoScaleManager.scale = scale;
+            const CGRect expectedRect = CGRectMake(0, 0, roundf(testScreenSize.width/scale), roundf(testScreenSize.height/scale));
+            const CGRect testRect = videoScaleManager.appViewportFrame;
```

Comment:
I don't think this test is needed as the RPC spec says that scale has a min value of 1.0

https://github.com/smartdevicelink/rpc_spec/blob/8b8c996c738711c4235c8574c179354cbf8982e2/MOBILE_API.xml#L3293